sdk/core/src/services/WalletStorageService.ts

```diff
@@ -293,93 +94,100 @@ export default class WalletStorageService {
     return signedCredentials as SignedCredential[]
   }
 
-  private static _getPaginationOptionsWithDefault(
-    fetchCredentialsPaginationOptions?: FetchCredentialsPaginationOptions,
-  ): PaginationOptions {
-    const { skip, limit } = fetchCredentialsPaginationOptions || {}
+  public async saveCredentials(credentials: any[], storageRegion?: string) {
+    storageRegion = storageRegion || this._storageRegion
 
-    return {
-      skip: skip || 0,
-      limit: limit || 100,
-    }
+    const responses = await this._affinidiVaultStorageService.saveCredentials(credentials, storageRegion)
+
+    return responses
   }
 
-  async encryptAndSaveCredentials(data: unknown[], storageRegion?: string) {
-    const publicKeyBuffer = this._keysService.getOwnPublicKey()
-    const encryptedCredentials = []
+  public async getAllCredentials(types?: string[][], storageRegion?: string) {
```

Comment:
But it's not "all", it's more like `getCredentialsByTypes` (similar to `getCredentialsByToken`).

I'd suggest to have three methods:
* `getAllCredentials(storageRegion)`
* `getCredentialsByTypes(types, storageRegion)`, with required types parameter
* `getCredentialsByToken(token, storageRegion)`, with required token parameter (that is, without handling a special case of token being empty)

Reply:
I was trying make it, to be similar to Affinidi vault api, but more logically would be separate them for single responsibility.